c2corg_api/views/validation.py

```diff
@@ -333,7 +331,10 @@ def _add_associations(
 
 
 def _is_parent_of_association(main_document_type, other_document_type):
-    if (main_document_type, other_document_type) in valid_associations:
+    # book is always main_d_t=parent?
+    if main_document_type == 'b' and other_document_type in valid_associations:
```

Comment:
Ok, my mistake.... I think that now we have no association between Book and Book...?https://github.com/c2corg/camptocamp.org/blob/master/data/sql/tables/db_associations.sql#L20-L71